libs/template/templates/default-python/databricks_template_schema.json

```diff
@@ -29,6 +29,14 @@
             "enum": ["yes", "no"],
             "description": "Include a stub (sample) Python package in '{{.project_name}}{{path_separator}}src'",
             "order": 4
+        },
+        "serverless": {
+            "type": "string",
+            "default": "no",
+            "enum": ["yes", "no", "auto"],
+            "description": "Use serverless compute. If auto, it will be enabled unless it is disabled on workspace level.",
+            "order": 5,
+            "skip_prompt_if": {}
```

Comment:
So this would always skip the prompt? Could you add a comment? (which is probably best done by adding a `"comment"` property inside the `"skip_prompt_if"`)